### DISCUSSION TITLE
How to suppress AbortError: Fetch is aborted ?

### DISCUSSION THREAD

**Discussion by lilouartz** [remix-run/remix#9835] [Q&A]
I am seeing a lot of "AbortError: Fetch is aborted" errors logged in Sentry.

I am assuming these are just people cancelling the request midway.

<img width="1451" alt="Screenshot 2024-08-06 at 10 05 56 AM" src="https://github.com/user-attachments/assets/c581aa4c-18fb-4d7e-a38c-cb49b1d3d8c7">

<img width="1473" alt="Screenshot 2024-08-06 at 10 06 12 AM" src="https://github.com/user-attachments/assets/17980338-0328-4ab7-a630-b728346f00ce">

How do I filter them out?

I can just add them to `ignoreErrors` in Sentry, but I am wondering if I am overlooking something.

**Comment by NubeBuster** [comment]
I would like to know as well
